Cargo.toml

```diff
@@ -8,3 +8,14 @@ members = [
     "polars/polars-arrow",
     "py-polars",
 ]
+
```

Comment:
Could this be removed as this involves the whole workspace?

Reply:
This should indeed be set `cargo/config.toml` for py-polars for release builds.

Reply:
Up to you. There are two things that lead to this change:
- You had the profile.release in py-polars Cargo.toml, where it had no effect (as cargo workspaces share settings, as they share compilation cache etc.)
- You had target-cpu in Cargo.toml, where it has no effect (it's a config.toml setting)

If you want to have different settings for py-polars (and leave defaults for workspace), I think this would mean to remove py-polars from the workspace. This would make sense to me, as then the pure Rust project would be split from the Python bindings.

Advantage would be: no LTO in testing (I did have one LLVM OOM on Windows in CI).
Disadvantage (depending on your point of view): when developing, build twice.

Not sure about what to do with the target-cpu "native", maybe best to leave this out and let people compile with it on via RUSTFLAGS optionally.

Reply:
Yes, I was under the impression that I move it to a `config.toml`, but that did not make it into master it seems. 

I think that removing py-polars from the workspace is best indeed. 

> BTW, you can mark several lines to comment on, which makes it easier to figure out what you're referring to :)

Noted ;)